artifactory/commands/repository/repository.go

```diff
@@ -108,7 +108,8 @@ var writersMap = map[string]utils.AnswerWriter{
 	YumRootDepth:                      utils.WriteIntAnswer,
 	DockerApiVersion:                  utils.WriteStringAnswer,
 	EnableFileListsIndexing:           utils.WriteBoolAnswer,
-	OptionalIndexCompressionFormats:   utils.WriteStringAnswer,
+	OptionalIndexCompressionFormats:   utils.WriteStringArrayAnswer,
+        PrimaryKeyPairRef:                 utils.WriteStringAnswer,
```

Comment:
Although this issue isn't visible in this GitHub view, this new line isn't properly indented. Running `go fmt ./...` while at the root of the codebase will fix this issue.

<img width="833" alt="image" src="https://github.com/jfrog/jfrog-cli-core/assets/7830056/e5c53425-7fac-4770-a519-daf023993aa4">

The same goes for the following lines in `template.go` - 

1.
<img width="903" alt="image" src="https://github.com/jfrog/jfrog-cli-core/assets/7830056/ec79306d-69e7-4cb2-a210-c6e6ac260b4c">



2,
<img width="1034" alt="image" src="https://github.com/jfrog/jfrog-cli-core/assets/7830056/489d3853-cb5a-4e18-9f39-22293f287951">



3.
<img width="690" alt="image" src="https://github.com/jfrog/jfrog-cli-core/assets/7830056/c0533d41-dba2-47bb-8283-f8fd21ea8604">